packages/components/src/components/link/_link.scss

```diff
@@ -60,7 +60,16 @@
     color: $visited-link;
   }
 
-  .#{$prefix}--link--disabled {
+  #{$prefix}--link--disabled,
+  #{$prefix}--link--disabled:focus,
+  #{$prefix}--link--disabled:hover,
+  #{$prefix}--link--disabled:active,
+  #{$prefix}--link--disabled:visited,
```

Comment:
out of curiousity, these don't have the `.` before it so not a class selector.  Was that intentional?